cirq-google/cirq_google/devices/google_noise_properties.py

```diff
@@ -0,0 +1,137 @@
+# Copyright 2022 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Class for representing noise on a Google device."""
+
+from dataclasses import dataclass, field
+from typing import Dict, List, Set, Type
+import numpy as np
+
+import cirq, cirq_google
+from cirq import _compat, devices
+from cirq.devices import noise_utils
+from cirq.transformers.heuristic_decompositions import gate_tabulation_math_utils
+
+
+SINGLE_QUBIT_GATES: Set[Type['cirq.Gate']] = {
+    cirq.ZPowGate,
+    cirq.PhasedXZGate,
+    cirq.MeasurementGate,
+    cirq.ResetChannel,
+}
+SYMMETRIC_TWO_QUBIT_GATES: Set[Type['cirq.Gate']] = {
+    cirq_google.SycamoreGate,
+    cirq.FSimGate,
+    cirq.PhasedFSimGate,
+    cirq.ISwapPowGate,
+    cirq.CZPowGate,
+}
+ASYMMETRIC_TWO_QUBIT_GATES: Set[Type['cirq.Gate']] = set()
+
+
+@dataclass
+class GoogleNoiseProperties(devices.SuperconductingQubitsNoiseProperties):
+    """Noise-defining properties for a Google device.
+
+    Inherited args:
+        gate_times_ns: Dict[type, float] of gate types to their duration on
+            quantum hardware.
+        t1_ns: Dict[cirq.Qid, float] of qubits to their T_1 time, in ns.
+        tphi_ns: Dict[cirq.Qid, float] of qubits to their T_phi time, in ns.
+        readout_errors: Dict[cirq.Qid, np.ndarray] of qubits to their readout
+            errors in matrix form: [P(read |1> from |0>), P(read |0> from |1>)].
+        gate_pauli_errors: dict of noise_utils.noise_utils.OpIdentifiers (a gate and the qubits it
```

Comment:
Can you clarify how `gate_pauli_errors` will be used? Is this the pauli errors of the depolarizing channels added after each gate? What determines the actual added pauli error for the depolarizing channel? How is this computed?

Reply:
(Probably this should be discussed in the parent class).